### PR TITLE
VZ-3516: Create a Jenkins script for post PRT steps

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

Includes a Jenkins script to call the scripts added for post PRT steps. The scripts added earlier were updated, as appropriate.

Fixes VZ-3516

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: release/builds/JenkinsfilePostPRT [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                }
+            }
+        }
+
+        stage('Scan Release Binaries') {
+            environment {
+                SCANNER_ARCHIVE_LOCATION = credentials('scanner-archive-location')
+                SCANNER_ARCHIVE_FILE = credentials('scanner-archive-file')
+                NO_PROXY_SUFFIX = credentials('cdn-no-proxy')
+                VIRUS_DEF_LOCATION = "https://update.nai.com/products/commonupdater2/current/vscandat1000/dat/0000"
```
Comment (agv426) [inline release/builds/JenkinsfilePostPRT:107]: Should we provide this as a global environment setting, or at least set it once in the environment up top if multiple stages need it?

Comment (agv426) [reply release/builds/JenkinsfilePostPRT:107]: ps, can tweak in the next pass here if we decide to change it

File: release/builds/JenkinsfilePostPRT [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                    cd ${WORKSPACE}/release/scripts
+                    ./create_github_release.sh ${params.RELEASE_VERSION} ${params.RELEASE_COMMIT} ${RELEASE_BINARIES_DIR} ${TEST_RUN}
+                  """
+                }
+            }
+        }
+    }
+    post {
+        always {
+            archiveArtifacts artifacts: "**/scan_report.out,**/scan_summary.out,", allowEmptyArchive: true
```
Comment (agv426) [inline release/builds/JenkinsfilePostPRT:145]: wondering if we should save the scan report/summary in Object storage as well? (again can be a followup if we think we should)

**Review by agv426 — APPROVED** [review]

File: release/builds/JenkinsfilePostPRT (hunk 1 above)
Comment (bhatpmk) [reply release/builds/JenkinsfilePostPRT:107]: We need this only for this stage, so an environment variable is defined here. We can define it as a global environment setting.

File: release/builds/JenkinsfilePostPRT (hunk 2 above)
Comment (bhatpmk) [reply release/builds/JenkinsfilePostPRT:145]: Yes, we can do that if that is useful.

**Pull request merged**